content/docs/reference/glossary/index.md

```diff
@@ -7,5 +7,6 @@ aliases:
   - /glossary
   - /docs/welcome/glossary.html
   - /docs/reference/glossary.html
+  - /help/glossary.html
```

Comment:
This should just be /help/glossary/

The use of .html was with the original Jekyll-based version of the site.